RapidReact/src/main/java/frc/robot/commands/AutoShootCommand.java

```diff
@@ -34,27 +34,81 @@ public AutoShootCommand withParameters(int ballCount, boolean top)
 
     private Command getSingleBallShootCommand()
     {
-        return new InstantCommand(this.top ? () -> this.shooter.spinUpTop() : () -> this.shooter.shootLow()) //Activate shooter
-                .andThen(new WaitCommand(0.3) //new WaitUntilCommand(() -> this.shooter.isUpToSpeed()) //Wait until shooter is up to speed
-                        .andThen(() -> this.rgb.autoShootingSingle()) //Very important RGB
-                        .andThen(this::enableFeedersBMS) //Feed ball
-                        .andThen(new WaitCommand(0.2) //Wait a bit, then turn off everything (how long it takes ball to shoot)
-                                .andThen(this::disableShooterFeedersBMS)));
+        SequentialCommandGroup command = new SequentialCommandGroup();
+
+        command.addCommands(
+
+                //Activate shooter
+                new InstantCommand(this.top ? () -> this.shooter.spinUpTop() : () -> this.shooter.shootLow()),
+
+                //Wait for shooter to get up to speed
+                new WaitCommand(0.3), //new WaitUntilCommand(() -> this.shooter.isUpToSpeed())
```

Comment:
we should use isUpToSpeed here.

Reply:
Oh yeah it should be more accurate now right

Reply:
yep!